doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -120,9 +120,27 @@ PSI Variables by Alpha
    energy [H] and correlation correction components [H] for the compound
    method requested through cbs().
 
+.. psivar:: CC ROOT n DIPOLE X
```

Comment:
Oh, and the switching `DIPOLE` to `QUADRUPOLE` was right? That's the edit that particularly concerned me.